src/shared/services/document-management/document-management.service.ts

```diff
@@ -26,4 +31,40 @@ export class DocumentManagementService {
         map(response => response.json())
       );
   }
+
+  createMediaViewer(caseField: CaseField): string {
+    let mediaViewer = {};
+    if (caseField.value) {
+      mediaViewer = {
+        document_binary_url: this.transformDocumentUrl(caseField.value.document_binary_url),
+        document_filename: caseField.value.document_filename,
+        content_type: this.getContentType(caseField),
+      }
+    }
+    return JSON.stringify(mediaViewer);
+  }
+
+  getContentType(caseField: CaseField): string {
+    let fileExtension = '';
+    if (caseField.value && caseField.value.document_filename) {
```

Comment:
`caseField.value` check redundant, it has already been checked in `createMediaViewer` method

Reply:
Removed the check.